src/components/text/text.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+
+import './text.css';
+
+import data from '../../data';
+
+const Text = ({ pageName, controlHeader }) => {
+  const pageData = data[pageName];
+  let dataKey = '';
+
+  switch (controlHeader) {
```

Comment:
instead of switch you can use
```code
 const keyMap = {
   what: 'What?',
   how: 'How?',
   more-information: 'More information?'
 };
 const dataKey = keyMap[controllerHeader];
```
switch is also good, but just be noticed that you can use a kind of map for such cases
make sense?

Reply:
Looks much more concise. Thanks!

Reply:
Done!